moz_security/sandboxes/heka/analysis/moz_security_cloudtrail.lua

```diff
@@ -119,9 +119,14 @@ function process_message()
             end
 
             if det[field[1]] then
-                if string.match(det[field[1]], field[2]) then
+              -- All items in the event field after the first one are potential matches
+              local i = 2
+              while field[i] do
+                if string.match(det[field[1]], field[i]) then
                     match_counter = match_counter + 1
```

Comment:
Should break after the first successful match (saves some processing and prevents match_counter from potentially being off if multiple items matched)